wallet/lib/pages/purchase_item/purchase_item_view_model.dart

```diff
@@ -441,8 +442,17 @@ class PurchaseItemViewModel extends ChangeNotifier {
     notifyListeners();
   }
 
-  void shareNFTLink(Size size, String link) {
-    shareHelper.shareText(text: link, size: size);
+  Future<void> shareNFTLink({required Size size}) async {
```

Comment:
Put walletstore from the constuctor.  And Kindly add test for this.

Reply:
Suggestions resolved. We will definitely add test cases once we are done with the base test case structure.